testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/transaction/mdb/timeout/TransactionTimeoutQueueSetupTask.java

```diff
@@ -83,21 +83,25 @@ public void tearDown(ManagementClient managementClient, String containerId) thro
 
     // /subsystem=messaging-activemq/server=default/address-setting=queue.propertyTimeoutQueue:add(max-delivery-attempts=1)
     private void setMaxDeliveryAttempts(String queueName) throws IOException {
-        ModelNode address = amqServerDefaultAdress.add("address-setting", "queue." + queueName);
-
-        ModelNode operation = new ModelNode()
-            .get(ClientConstants.OP_ADDR).set(address)
-            .get(ClientConstants.OP).set(ClientConstants.ADD)
-            .get("max-delivery-attempts").set(1);
+        ModelNode address = amqServerDefaultAdress.clone().add("address-setting", "jms.queue." + queueName);
+
+        ModelNode operation = new ModelNode();
+        operation.get(ClientConstants.OP).set(ClientConstants.ADD);
+        operation.get(ClientConstants.OP_ADDR).set(address);
+        operation.get("max-delivery-attempts").set(1);
+        // delay redelivery of messages for 5s
+        operation.get("redelivery-delay-multiplier").set(1.5);
```

Comment:
thank you for correcting me in wrongly put fluent calls of the `ModelNode().get(...).set(..)...` which do not work!

from my testing I think the `max-delivery-attempts` could be enough. the others do not add much different behaviour.